tests/integration/mock/eventstream_runtime/CodeActAgent/test_ipython/prompt_002.log

```diff
@@ -411,5 +411,6 @@ with open('/workspace/test.txt', 'w') as file:
 OBSERVATION:
 [Code executed successfully with no output]
 [Jupyter current working directory: /workspace]
+[Jupyter Python interpreter: /opendevin/poetry/opendevin-w8YlaYp7-py3.11/bin/python]
```

Comment:
`w8YlaYp7` sounds like non-deterministic. If tests indeed fail, we need to apply a hack/fix to integration testing to ignore certain known non-deterministic patterns.

Reply:
I think this hash is calculated based on the absolute path of the `/opendevin/code`, which should be deterministic (until we change it). But let's see if it passes all the test!

https://github.com/python-poetry/poetry/issues/3459#issuecomment-748618616